packages/client/src/pages/FindAMentor/Mentor.js

```diff
@@ -1,26 +1,59 @@
-import React, { useState } from "react";
+import React from 'react';
```

Comment:
this component is for the modal here right:
![image](https://user-images.githubusercontent.com/12561777/87734746-1daca980-c799-11ea-94a0-40782193c2fb.png)


can we also add the individual grid items to this PR:
![image](https://user-images.githubusercontent.com/12561777/87734759-2a310200-c799-11ea-9bc5-45ea515d8c2e.png)

designs here: https://www.figma.com/file/3IPF991kTgpLJ3LDyyzAgL/CovEd-Web-Redesign?node-id=290%3A1715

Reply:
Wait I don't think so. That is the Mentor Request Frame. The mentor card is just the initial profiles that will be placed within the grid.

Reply:
I think you're right however I was working on trying to make something like this:
![Screen Shot 2020-07-16 at 10 58 52 PM](https://user-images.githubusercontent.com/32288394/87747147-0d57f700-c7b8-11ea-85b7-1a7dc5392348.png)

So you can effectively use the mentor component that's in this PR for that

Reply:
Ah ok. So you made the profile page? Then I can probably just plop it into a modal frame with minor adjustments. Do you want to rename the this PR lol?

Reply:
I'll finish the actual cards and add it to this PR I think